centaur/test_cromwell.sh

```diff
@@ -124,11 +124,11 @@ cd "${RUN_DIR}"
 TEST_STATUS="failed"
 
 if [[ "${CENTAUR_SBT_COVERAGE}" == "true" ]]; then
-    sbt --error coverage centaur/it:compile
-    CP=$(sbt --error coverage "export centaur/it:dependencyClasspath" -error)
+    sbt --warn coverage centaur/it:compile
+    CP=$(sbt --warn coverage "export centaur/it:dependencyClasspath" -error)
 else
-    sbt --error centaur/it:compile
-    CP=$(sbt --error "export centaur/it:dependencyClasspath" -error)
+    sbt --warn centaur/it:compile
+    CP=$(sbt --warn "export centaur/it:dependencyClasspath" -error)
```

Comment:
FYI This uses double-dash `warn` and then later single-dash `error` on the same line. Btw, double-dashes [weren't working for a while](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Fsbt%2Fsbt%2Fissues%2F4088).